src/Microsoft.OData.Edm/EdmUtil.cs

```diff
@@ -492,6 +500,82 @@ internal static string FullyQualifiedName(IEdmVocabularyAnnotatable element)
             return null;
         }
 
+        internal static string FullyQualifiedName(IEdmAnnotationsTarget annotationsTarget)
+        {
+            string qualifiedName = string.Empty;
+            List<IEdmElement> segments = annotationsTarget.TargetSegments.ToList();
+            IEdmEntityContainer container = segments[0] as IEdmEntityContainer;
+            IEdmEntityContainerElement containerElement = segments[1] as IEdmEntityContainerElement;
+
+            if (container == null || containerElement == null)
+            {
+                return null;
+            }
+
+            qualifiedName = container.FullName() + "/" + containerElement.Name;

```

Comment:
The compiler will most likely convert this into `string.Concat` but I'd be happier if you used `string.Concat`.